content/integrations/sumologic.md

```diff
@@ -13,11 +13,12 @@ Enable the Sumo Logic integration to:
 
 ## Installation
 
-There are two parts to the SumoLogic integration, based on the direction of information between the products. You can send alerts and events from Datadog to Sumologic and you can have SumoLogic send data to Datadog.
+Datadog integration with SumoLogic is two-part: send alerts and events from Datadog to Sumologic, or have SumoLogic send data to Datadog. You can enable one or both.
```

Comment:
that would be better i think. but most will use both, kinda like in pagerduty. Although im not sure what was wrong with the original.